src/controller/AutoCommissioner.cpp

```diff
@@ -777,17 +785,76 @@ CHIP_ERROR AutoCommissioner::CommissioningStepFinished(CHIP_ERROR err, Commissio
                     mParams.ClearICDStayActiveDurationMsec();
                 }
             }
+
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+            if (mParams.GetExecuteJCM().ValueOr(false) &&
+                (mDeviceCommissioningInfo.JFAdministratorFabricIndex != kUndefinedFabricIndex))
+            {
+                if (!mJFAdminRCAC.Alloc(mDeviceCommissioningInfo.JFAdminRCAC.size()))
+                {
+                    ChipLogError(Controller, "AutoCommissioner: cannot allocate memory for mJFAdminRCAC");
+                    return CHIP_ERROR_NO_MEMORY;
+                    ;
+                }
+                memcpy(mJFAdminRCAC.Get(), mDeviceCommissioningInfo.JFAdminRCAC.data(), mJFAdminRCAC.AllocatedSize());
+                mParams.SetJFAdminRCAC(ByteSpan(mJFAdminRCAC.Get(), mJFAdminRCAC.AllocatedSize()));
```

Comment:
```suggestion
                mParams.SetJFAdminRCAC(mJFAdminRCAC.Span());
```

and similar elsewhere?

Reply:
Applied all these suggestions, resolving this conversation.